Module4_Labs/Lab6_Sudoku_Solver/Cards/5.md

```diff
@@ -31,9 +31,8 @@ def nextMove(grid,silent=False,depth=0,finish=False):
 
 Now for defining this function, we will be using it in conjunction with our `findPossible()` function which we defined previously. To provide insight on the structure of this function, we will break it down into three possible clauses:
 
-- 1) An easy iteration through all spots, checking if clearly only one value can exist for a particular square.
-- 2) Checking by number: within each number we will check by row, column and box.
-- 3) Guessing clause which will be the fallback guessing and checking mechinism (this segment uses a function called `testPossible()` which we have not yet defined)
+- 1) An easy iteration through all spots: within each number we will check by row, column and box.
```

Comment:
2 shouldn't be removed- it should just be explained better what the difference between 1 and 2 are. They use different code so it would screw up the whole lab if removed